gcloud/Dockerfile

```diff
@@ -17,8 +17,9 @@ RUN apt-get -y update && \
         --bash-completion=false \
         --disable-installation-options && \
 
-    # Install alpha and beta components
-    /builder/google-cloud-sdk/bin/gcloud -q components install alpha beta && \
+    # Install additional components
+    /builder/google-cloud-sdk/bin/gcloud -q components install \
+        alpha beta kubectl && \
```

Comment:
Help me understand why we install kubectl in the gcloud builder rather than just adding it to the kubectl builder?